kse/src/org/kse/gui/dialogs/resources.properties

```diff
@@ -565,3 +565,4 @@ RevokedCertsTableModel.SerialNumberColumn   = Serial Number
 DCompareCertificates.Title          = Compare Certificates
 DCompareCertificates.jbOK.text      = Ok
 DCompareCertificates.KeyUsage       = Key Usage: {0}
+DCompareCertificates.jlMatch.text   = % of matching: {0}
```

Comment:
Hi @jgrateron,

Here is a minor request...

Perhaps change:
- `% of matching: {0}`
to 
- `Matching: {0}%`

_to debate..._ 

Ref.: https://en.wikipedia.org/wiki/Percent_sign

_[I can prepare the French resources...]_

Reply:
great, much better, thanks